packages/editor/src/client/routes/contentEditor.tsx

```diff
@@ -0,0 +1,537 @@
+        duration: 2000
+      })
+    } else {
+      const {data} = await createContent({variables: {input}})
+
+      if (data) {
+        dispatch({
+          type: RouteActionType.ReplaceRoute,
+          route: ContentEditRoute.create({type, id: data.content[type].create.id})
+        })
+      }
+      setChanged(false)
+      Notification.success({
+        title: t('articleEditor.overview.draftCreated'),
+        duration: 2000
+      })
+    }
+  }
+
+  async function handlePublish(publishDate: Date, updateDate: Date) {
+    if (contentdId) {
+      const {data} = await updateContent({
+        variables: {id: contentdId, input: createInput()}
+      })
+
+      if (data) {
+        const {data: publishData} = await publishContent({
+          variables: {
+            id: contentdId,
+            publishAt: publishDate.toISOString(),
+            publishedAt: publishDate.toISOString(),
+            updatedAt: updateDate.toISOString()
+          }
+        })
+
+        if (publishData?.content?._all?.publish?.published?.publishedAt) {
+          setPublishedAt(new Date(publishData?.content?._all?.publish?.published.publishedAt))
+        }
+      }
+
+      setChanged(false)
+      Notification.success({
+        title: t('articleEditor.overview.articlePublished'),
+        duration: 2000
+      })
+    }
+  }
+
+  useEffect(() => {
+    if (isNotFound) {
+      Notification.error({
+        title: t('articleEditor.overview.notFound'),
+        duration: 5000
+      })
+    }
+  }, [isNotFound])
+
+  let content = null
+  if (contentConfig.getContentView) {
+    content = contentConfig.getContentView(contentData, handleChange, isLoading || isDisabled)
+  } else {
+    content = (
+      <GenericContentView
+        record={contentData}
+        model={contentConfig.schema.content}
+        dispatch={dispatcher}></GenericContentView>
+    )
+  }
+
+  let metadataView = null
+  if (contentConfig.getMetaView) {
+    metadataView = contentConfig.getMetaView(
+      metadata,
+      customMetadata,
+      value => {
+        setMetadata(value)
+        setChanged(true)
+      },
+      (value: any) => {
+        setCustomMetadata(value)
+        setChanged(true)
+      }
+    )
+  } else {
+    metadataView = (
+      <ContentMetadataPanel
+        defaultMetadata={metadata}
+        onChangeDefaultMetadata={(value: any) => {
+          setMetadata(value)
+          setChanged(true)
+        }}
+      />
+    )
+  }
+
+  return (
+    <>
+      <EditorTemplate
+        navigationChildren={
+          <NavigationBar
+            leftChildren={
+              <IconButtonLink
+                size={'lg'}
+                appearance="subtle"
+                icon={<Icon icon="angle-left" />}
+                route={ContentListRoute.create({type})}
+                onClick={e => {
+                  if (!unsavedChangesDialog()) e.preventDefault()
+                }}>
+                {t('articleEditor.overview.back')}
```

Comment:
i would avoid using translation strings from `articleEditor`. Also i think some of the translations in here are available in the globals.